main.go

```diff
@@ -91,6 +91,11 @@ func parseTime(raw string) (t time.Time, err error) {
 		return t, nil
 	}
 
+	t, err = time.Parse("2006-01-02", raw)
+	if err == nil {
```

Comment:
and not like me, you got the `==` right 😉  👍

Reply:
By copying and pasting ... 😆